cmd/kcp/kcp.go

```diff
@@ -68,18 +68,26 @@ func main() {
 
 	// manually extract root directory from flags first as it influence all other flags
 	rootDir := ".kcp"
+	additionalMappingsFile := ""
 	for i, f := range os.Args {
 		if f == "--root-directory" {
 			if i < len(os.Args)-1 {
 				rootDir = os.Args[i+1]
 			} // else let normal flag processing fail
 		} else if strings.HasPrefix(f, "--root-directory=") {
 			rootDir = strings.TrimPrefix(f, "--root-directory=")
+		} else if f == "--mapping-file" {
+			if i < len(os.Args)-1 {
+				additionalMappingsFile = os.Args[i+1]
+			} // else let normal flag processing fail
+		} else if strings.HasPrefix(f, "--mapping-file") {
+			additionalMappingsFile = strings.TrimPrefix(f, "--mapping-file=")
```

Comment:
One is for front-proxy in production where we use this. 
Another is at the server level for cases when we don't have a proxy. In this case when we run slim mini proxy with `kcp start` and we need to pass the flag in somehow. 

maybe we can rename one at the server level to something `dev-mapping-file` so it's not confusing?

Reply:
But this PR doesn't touch the option in `pkg/proxy/options/options.go`, which is a separate implementation of `--mapping-file`. I think a `--miniproxy-mapping-file` flag on the server would be nicer, but I'm still not sure I understand why we read the flag in two places 😅

Reply:
Because we need to load it into fake-frontproxy :) So if we rename it will be cleaner - one for frontproxy, another for minifrontproxy.

Reply:
So TL:DR: 
its 2 different kube apiservers - proxy and core kcp server. Core server should never have this flag defined. But because we share code with `kcp start` I didn't find nice way not to add it. Not its implied its development only.